docs/connector-development/connector-builder-ui/partitioning.md

```diff
@@ -12,67 +12,69 @@ There are some cases that require multiple requests to fetch all records as well
 * If your records are spread out across multiple pages that need to be requested individually if there are too many records, use the Pagination feature.
 * If your records are spread out over time and multiple requests are necessary to fetch all data (for example one request per day), use the Incremental sync feature.
 
-## Dynamic and static partition routing
+## Dynamic and static partitioning
 
-There are three possible sources for the partitions that need to be queried - the connector itself, supplied by the end user when configuring a Source based on the connector, or the API  provides the list of partitions on another endpoint (for example the Woocommerce API also includes an `/orders` endpoint that returns all orders).
+There are three possible sources for the partitions that need to be queried - the connector itself, supplied by the end user when configuring a Source based on the connector, or the API provides the list of partitions on another endpoint (for example the Woocommerce API also includes an `/orders` endpoint that returns all orders).
```

Comment:
Totally optional side suggestion:
```suggestion
There are three possible sources for the partitions that need to be queried - the connector itself, supplied by the end user when configuring a Source based on the connector, or the API provides the list of partitions via another endpoint (for example the Woocommerce API also includes an `/orders` endpoint that returns all orders).
```

I do think "via" is marginally better, but the real reason for the suggestion is that I had to reread the sentence to make sure it was coherent purely because "or" and "on" are so visually similar.